### PR TITLE
Add hotfixes for 1.20.7 and 1.21.2

### PR DESCRIPTION
**Pull request by ganeshkumar5699**
Adding latest CVE fixes.

### CODE REVIEW

File: vhdbuilder/packer/test/linux-vhd-content-test.sh [hunk 1]
```diff
@@ -205,8 +205,10 @@ testKubeBinariesPresent() {
   1.19.13
   1.20.2-hotfix.20210310
   1.20.5-hotfix.20210322
+  1.20.7-hotfix.20210816
```
Comment (alexeldeib) [inline vhdbuilder/packer/test/linux-vhd-content-test.sh:208]: so we weren't previously testing either of these versions in vhd content test? oops

**Comment by alexeldeib** [comment]
lgtm pending the make issue? the error output doesn't have much, might need to check the make target and manually debug it a tiny bit

File: vhdbuilder/packer/test/linux-vhd-content-test.sh (hunk 1 above)
Comment (ganeshkumar5699) [reply vhdbuilder/packer/test/linux-vhd-content-test.sh:208]: Yes, we were not and I was surprised by that too.

**Comment by ganeshkumar5699** [comment]
> lgtm pending the make issue? the error output doesn't have much, might need to check the make target and manually debug it a tiny bit

That was a transient issue. Checked with @anujmaheshwari1 and re-ran.

**Review by alexeldeib — APPROVED** [review]
Lgtm

**Pull request merged**